### ISSUE TITLE
pubsublite: better document on how to use EventTime - in Publish and Seek operation

### ISSUE THREAD

**Issue by rnadhani-ns** [googleapis/google-cloud-go#7843]
**Client**

PubSubLite

**Environment**

Mac Os X

**Go Environment**

pubsublite v1.6.0

go version
go version go1.20.1 darwin/amd64

**Code**

A fairly vanilla code to publish
```go
publishedMessage <- publisher.Publish(pubSubCtx, &pubsub.Message{
  Data:       loggable,
  Attributes: attributeMap,
})
```

I am reading: https://cloud.google.com/pubsub/lite/docs/publishing and the URL link: https://cloud.google.com/pubsub/lite/docs/publishing#setting_the_event_timestamp_attribute to set event_timestamp seems to go nowhere.

**Expected behavior**

I should be able to understand on how to set EventTime to a message by reading some examples in the doc.

**Actual behavior**

Not able to use `pubsublite.EventTime(<time>):` as described at: https://cloud.google.com/pubsub/lite/docs/seek. I can very well use Beginning, End. Publishtime but unable to understand ot use EventTime

**Screenshots**

N;/A

**Additional context**

Basically, how do I set eventtimestamp when publishing my message. I could even use OrderingKey feature. Just unable to figure out how to set the value during publishing. The only reference i saw was from: https://github.com/googleapis/google-cloud-go/blob/main/pubsublite/pscompat/message.go#L34 but is that the attribute key that we need? What is the format of the string of that attribute?

**Comment by tmdiep** [comment]
Hi @rnadhani-ns, it just so happens that we are currently working on samples for this. I will update once they are available. ETA by the end of this week.

**Comment by rnadhani-ns** [comment]
@tmdiep Thanks. I was also browing through the code and the other reference that i see was: https://github.com/googleapis/google-cloud-go/blob/ba0490ada08f052ace13954f7a1c72923a742c83/pubsublite/pscompat/message_test.go#L75 -> but that clearly is a test to convert from Wire to Golang struct object and not the other way around.

Looking forward to the docs.

**Comment by tmdiep** [comment]
Unfortunately, this is exactly how the event time is encoded as a message attribute. I can see that it is quite awkward if you wanted to set event timestamps for each message. Would a RFC3339 formatted timestamp be more usable?

Aside: Pub/Sub Lite implements the publisher and subscriber interfaces of Cloud Pub/Sub (the `pubsub` module), which does not contain a first-class field for the event time.

**Comment by tmdiep** [comment]
Please disregard the last comment. We will expose the event timestamp encoding and decoding functions.

**Comment by rnadhani-ns** [comment]
@tmdiep TBH, in the end it does not really matter what attribute name I have to set. We can always make it a constant in our code. Generally, through the pipelines the times are set in Unix epoch timestamp. So any method that can encode/decode from that and maybe even a library level Public constant that we can use to set as attribute name would be sufficient.

No need to be fancy :)

**Comment by tmdiep** [comment]
@rnadhani-ns We just cut pubsublite release 1.8.0, which exposes the encode/decode functions for the event time.

The godocs will take some time to update. In the meantime please see these examples: https://github.com/googleapis/google-cloud-go/blob/421053a70864063b127a60fa62914da05f84961d/pubsublite/pscompat/example_test.go#L226

I will close this issue. But we are planning to also update the documentation at https://cloud.google.com/pubsub/lite/docs/publishing to include samples for setting the event time. Feel free to reopen if you run into any issues.
